docs/basic/code/greedy/greedy_1.cpp

```diff
@@ -0,0 +1,38 @@
+#include <algorithm>
+#include <cmath>
+#include <cstdio>
+#include <cstring>
+#include <iostream>
+#include <queue>
+using namespace std;
+struct f {
+  long long d;
+  long long x;
+} a[100005];
+bool cmp(f A, f B) { return A.d < B.d; }
+priority_queue<long long, vector<long long>, greater<long long> >
+    q;  //小根堆维护最小值
+
+int main() {
+  long long n, i, j;
+  cin >> n;
+  for (i = 1; i <= n; i++) {
+    scanf("%d%d", &a[i].d, &a[i].x);
```

Comment:
```suggestion
    cin >> a[i].d >> a[i].x;
```

原来这行用 `%d` 读入 `long long` 数据报 warning 啦（

Reply:
没发现.jpg
之后的我会注意下的
主要是没开-Wall

Reply:
对了，好像看 CI 的 log 也能发现（

Reply:
> 对了，好像看 CI 的 log 也能发现（

我就是看 log 发现这个问题的（